Makefile

```diff
@@ -169,7 +169,7 @@ push: image
 .PHONY: test
 test: clean
 	@echo "running unit tests"
-	go test ./... -race -tags deadlock -coverprofile=coverage.txt -covermode=atomic
+	go test `go list ./... | grep -v test/e2e` -cover -race -tags deadlock
```

Comment:
can this be simplified with `go test ./pkg/... ` ? 
And also rest of parameters are needed to give us code cover report, I don't think we need to remove them.

Reply:
Done